addons/l10n_tr/models/account_tax.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    l10n_tr_code = fields.Char(string="Code", size=3)
+    l10n_tr_tax_type = fields.Selection(selection=[
+        ('normal', 'Normal'),
+        ('exception', 'Exception'),
+        ('withdraw', 'VAT Withdraw'),
+    ], default='normal', string="Exception",
+        help="Technical field showing that this tax will be applied with "
+             "TaxExemptionReasonCode tag or with WithholdingTaxTotal tag in EDI.")
```

Comment:
I agree the names are a little confusing, but I tried to keep the same name as the EDI format. 
withdrawal taxes re indeed applied as negative taxes here too but if you want to use a withdrawal (like a tax discount) you have to say your invoice is a Withdrawal invoice and it should be tagged differently in EDI
Exceptions have the same issue, if you use a 0% VAT or no VAT you have to tag the invoice for EDI. 

it could be applied to the tax_group, but the problem is tax groups are printed below the invoice and the exceptions shouldn't be there as a separate line, so exceptions are either in vat group (code is 0015) or not printed at all (in case of no VAT), so I couldn't declare a vat_group for them

Reply:
done